### PR TITLE
Jw gawb 1290 view attributes for single entity

### PR DESCRIPTION
**Pull request by jlmwhitt**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs [hunk 1]
```diff
@@ -57,12 +58,53 @@
...
                  "Copy from another workspace"]]))]])}])})
 
+(react/defc EntityAttributes
+  {:render (fn [{:keys [props state]}]
+      (let [workspace (:workspace props)
+            bucket-name (get-in workspace [:workspace :bucketName])
+            attributes (:attributes props)
+            entity-name (:selected-entity props)
+            entity-type (str (:selected-entity-type props))]
+      (when (not= entity-type ":sample_set")
```
Comment (helgridly) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs:69]: What's this about?

**Review by helgridly — COMMENTED** [review]
Alex will want to see this if he hasn't already.

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs (hunk 1 above)
Comment (jlmwhitt) [reply src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs:69]: There are no attributes for the sample sets, so we don't show attributes at all when looking at them

Comment (helgridly) [reply src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs:69]: That's not true - sample sets can have attributes, and we should show them!

**Comment by jlmwhitt** [comment]
need to add support for sample set attributes

**Comment by helgridly** [comment]
^ Sample sets (and set types in general) aren't a special case - they have attributes just the same as other entities. I'd guess all you need to do is to remove the conditional.

**Pull request closed without merging**